tests/stats/mda8/test_mda8.py

```diff
@@ -154,6 +153,27 @@ def test_rollingaverage(test_data, exp_ravg):
     assert all(ravg[0, :, 0] == pytest.approx(exp_ravg, abs=0, nan_ok=True))
 
 
+def test_rollingaverage_label():
+    """
+    Checks that the labels of rolling average is correct (we want it to be labeled based
+    on the "left-most" interval, ie. earliest measurement). This seems to be the
+    current behaviour of xarray (but not well documented) and this test should
+    detect if this behaviour changes in xarray in the future.
+    """
+    data = xr.DataArray(
+        [[[x] for x in range(24)]],
+        dims=["data_source", "time", "station_name"],
+        coords={"time": xr.date_range(start="2024-01-01 00:00", periods=24, freq="1h")},
+    )
+
+    ravg = _rolling_average_8hr(data)
+
+    assert np.all(
+        ravg.get_index("time")
+        == xr.date_range(start="2024-01-01 00:00", periods=24, freq="1h") + pd.Timedelta("8h")
+    )
```

Comment:
Can this be re-written without `np.all`?
Maybe you can reuse `data["time"]` like this
``` python
    assert ravg.get_index("time") == data["time"] + pd.Timedelta("8h").all()
```
but that looks very similar to the actual implementation,
perhaps something like this would be better
``` python
    assert set(data["time"]-ravg.get_index("time")) == {pd.Timedelta("-8h")}
```
or assert on the first and last time stamps
``` python
    assert date["time"].isel(time=0).dt.strftime("%F %R") == "2024-01-01 08:00"
    assert date["time"].isel(time=23).dt.strfmt("%F %R") == "2024-01-02 08:00"